### PR TITLE
Fixing, cleaning and new features

### PR DESCRIPTION
**Pull request by alistairholt**
- Add missing development dependencies
- Bring .rvmrc up-to-date with current conventions
- Add support for Bundler
- Add support for object inspection via awesome_print
- Fix warnings about calling #split on nil

### CODE REVIEW

File: .rvmrc [hunk 1]
```diff
@@ -1 +1,28 @@
...
+    return 1
+  fi
+fi
+
+if [[ $- == *i* ]] # check for interactive shells
+then
+    echo "Using: $(tput setaf 2)$GEM_HOME$(tput sgr0)" # show the user the ruby and gemset they are using in green
+else 
+	echo "Using: $GEM_HOME" # don't use colors in interactive shells
+fi
```
Comment (rsanheim) [inline .rvmrc:28]: Hey why would rvmrc need to be so complicated?

Also, I'm not sold on using a gemset, especially since bundler should manage the gems.

**Comment by rsanheim** [comment]
Thanks for the bundler addition, would like to merge that part.  The awesome print stuff is fine, too, though I haven't used it.  Long term I suppose there should be a more general way to define how log buddy prints stuff.

File: .rvmrc (hunk 1 above)
Comment (alistairholt) [inline .rvmrc:28]: I agree about the complexity, it's just what RVM generates by default. I personally use a gemset for each project I'm working on to keep things clean and avoid gem version issues but that's just me.

**Comment by alistairholt** [comment]
I'm currently using my fork and the awesome_print functionality is working really well for me.

**Comment by cldwalker** [comment]
I've pulled in use_awesome_print and the warnings fix. Thanks!

**Pull request closed without merging**